src/Avalonia.Controls/Primitives/ScrollBar.cs

```diff
@@ -257,6 +257,17 @@ protected override void OnKeyDown(KeyEventArgs e)
             }
         }
 
+        protected override void OnPointerWheelChanged(PointerWheelEventArgs e)
+        {
+            base.OnPointerWheelChanged(e);
+
+            // We need to handle pointer wheel event to allow scrolling with the pointer wheel. So we raise the event on the scrollviewer's presenter
+            if(!e.Handled && _owner?.Presenter is { } presenter)
+            {
+                presenter.RaiseEvent(e);
```

Comment:
By re-raising the same `RoutedEventArgs`, this will change the observable `RoutedEventArgs.Source` and `Route`, so parent listeners will have wrong information to act on (e.g. `Tunnel` mode in a middle of a `Bubble`).

I think a copy of the event should be raised instead to avoid this.

The original event probably needs to be marked as handled if the copy is handled.

Reply:
Update done.